src/Arcus.WebApi.Tests.Unit/Security/Authorization/JwtTokenAutorizationFilterTests.cs

```diff
@@ -56,6 +55,34 @@ public async Task GetHealthWithCorrectBearerToken_WithAzureManagedIdentityAuthor
             }
         }
 
+        [Fact]
+        public async Task GetHealthWithCorrectBearerToken_WithAzureManagedIdentityAuthorizationAndCustomClaims_ReturnsOk()
```

Comment:
We could benefit from some failure tests too. See how the header name, the amount of claims... affect the result.

Reply:
Let's do a follow-up PR for more tests!